src/client/modules/common/components/native/ui-native-base/components/FormButton.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, Text } from 'react-native';
+import { Button } from 'native-base';
+
+const FormButton = ({ children, ...props }) => {
```

Comment:
I guess this should be only `Button`. It is not specifically tied to `Form` component.

Reply:
ah I see you have another component named `Button`

Reply:
@mairh I needed to style the button a bit differently if in form, so I added another wrapper.